enginetest/queries/query_plans.go

```diff
@@ -3608,8 +3608,8 @@ inner join pq on true
 			" ├─ HashIn\n" +
 			" │   ├─ Eq\n" +
 			" │   │   ├─ mytable.i:0!null\n" +
-			" │   │   └─ 1 (bigint)\n" +
-			" │   └─ TUPLE(true (tinyint))\n" +
+			" │   │   └─ 1 (double)\n" +
```

Comment:
Suspicious, where does this come from

Reply:
Found it! The `IsSigned()` util function wasn't considering `gmstypes.Boolean` as signed anymore since it didn't exactly match `Int8`. Fixed now.